Sources/CreateAPI/Generator/Generator+Schemas.swift

```diff
@@ -687,10 +687,42 @@ extension Generator {
             if context.isPatch && isOptional && options.paths.makeOptionalPatchParametersDoubleOptional {
                 type = type.asPatchParameter()
             }
+
             var defaultValue: String?
             if options.entities.includeDefaultValues {
-                if type.isBool {
-                    defaultValue = (info?.defaultValue?.value as? Bool).map { $0 ? "true" : "false" }
+                func describeDefaultValue(_ anyValue: AnyCodable) -> String? {
+                    if let bool = anyValue.value as? Bool {
+                        return bool ? "true" : "false"
+                    } else if let int = anyValue.value as? Int {
+                        return "\(int)"
+                    } else if let uint = anyValue.value as? UInt {
+                        return "\(uint)"
+                    } else if let double = anyValue.value as? Double {
+                        return "\(double)"
+                    } else if let string = anyValue.value as? String {
+                        if let formatString = info?.formatString, !formatString.isEmpty {
+                            return nil
+                        }
+                        // Surround the value in quotes, otherwise the string is rendered improperly.
+                        return "\"\(string)\""
+                    } else if let array = anyValue.value as? [AnyCodable] {
+                        return "\(array.map(describeDefaultValue(_:)))"
+                    } else if let dictionary = anyValue.value as? [String: AnyCodable] {
+                        return "\(dictionary.mapValues(describeDefaultValue(_:)))"
+                    } else {
+                        return nil
+                    }
+                }
+
+                if let value = info?.defaultValue.flatMap(describeDefaultValue(_:)), !value.isEmpty {
+                    let bareValue = value.trimmingCharacters(in: .quotes)
```

Comment:
I'm not quite sure if I follow this properly? This is stripping `"` or `'` from the leading/trailing edges of the string but for what reason? Are we trying to escape strings?

Reply:
Essentially, if the value is a string, we need to wrap it in quotes for rendering. But because defaults correspond to rawValues for enums, in order to maintain that support we need to strip out the quotes so we can check if there is a matching enum. We might be able to do this in `describeDefaultValue` though.